tobias/utils/motifs.py

```diff
@@ -248,70 +278,52 @@ def as_string(self, output_format="pfm"):
 			Motif format ("pfm", "jaspar", "meme")
 		"""
 
-		bases = ["A", "C", "G", "T"]
 		out_string = ""
 
-		#Establish which output format
-		if output_format in ["pfm", "jaspar"]:
-			for motif in self:
-				out_string += ">{0}\t{1}\n".format(motif.id, motif.name)
-
-				#Convert counts to string
-				counts_type = float if float in [type(element) for element in motif.counts[0]] else int	#Estimate from first base counts
-				if counts_type == float:
-					counts_string = [["{0:.5f}".format(element) for element in row_counts] for row_counts in motif.counts]
-				else:
-					counts_string = [[str(element) for element in row_counts] for row_counts in motif.counts]
-					element_width = max(sum([[len(element) for element in row_counts] for row_counts in counts_string], []))	#Length of longest element to align to
-					counts_string = [[" " * (element_width - len(element)) + element for element in row_counts] for row_counts in counts_string]
-
-				#Write out to format
-				for i, base_counts in enumerate(counts_string):
-					out_string += "{0} [ {1} ]\n".format(bases[i], "  ".join(base_counts)) if output_format == "jaspar" else "  ".join(base_counts) + "\n"
-				out_string += "\n"
+		header = True
+		for motif in self:
+			out_string += motif.as_string(output_format=output_format, header=header)
+			header = False

```

Comment:
Could you just add a comment here explaining that the header is written for the first motif, and that this is relevant only for meme? I know it says in as_string, but just for reference